### PR TITLE
device support for SM309 and TS110F

### PR DESCRIPTION
**Pull request by umairyounus**

### CODE REVIEW

File: docs/devices/S1.md [hunk 1]
```diff
@@ -21,6 +21,24 @@ description: "Integrate your Ubisys S1 via Zigbee2MQTT with whatever smart home
 ### Configuring Inputs
 In case the inputs need to be reconfigured (e.g. to use stationary switches instead of momentary ones) this can be done in the same way as [it is being done for the ubisys C4](C4.html#configuring-inputs).
 
+### Decoupling
```
Comment (Koenkk) [inline docs/devices/S1.md:24]: I see some unexpected changes here, can you undo them?

**Comment by umairyounus** [comment]
that's very odd, sorry let me fix it.

**Comment by umairyounus** [comment]
I have reset the changes, `npm run docgen`  is creating these side effects. I'm sure I'm doing something wrong. @Koenkk could you please point me in the right direction.

**Comment by Koenkk** [comment]
Running docgen is not required for pr, sorry for the inconvenience.

**Pull request merged**